### PR TITLE
Don't classify 'var' in 'out var' as a keyword.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/4190

### CODE REVIEW

File: src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs [hunk 1]
```diff
@@ -283,23 +283,7 @@ private static bool CouldBeVarKeywordInDeclaration(SyntaxToken token)
...
-                    if (token.Parent.Parent is ArgumentSyntax)
-                    {
-                        var argument = (ArgumentSyntax)token.Parent.Parent;
-                        if (argument.RefOrOutKeyword.IsKind(SyntaxKind.OutKeyword))
-                        {
-                            return true;
-                        }
-                    }
-                }
+                return token.Parent is IdentifierNameSyntax && token.Parent.Parent is ExpressionStatementSyntax;
```
Comment (Pilchie) [inline src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs]: Ah, nice.  We explicitly added this code for declaration expressions :smile:

**Comment by Pilchie** [comment]
:+1:

**Comment by dpoeschl** [comment]
:+1:

File: src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs (hunk 1 above)
Comment (DustinCampbell) [inline src/Workspaces/CSharp/Portable/Classification/ClassificationHelpers.cs]: ha. Oops! My fault there.

**Comment by DustinCampbell** [comment]
:+1:

**Comment by CyrusNajmabadi** [comment]
retest this please

**Comment by CyrusNajmabadi** [comment]
Fixes #4190

**Pull request merged**